drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -457,6 +457,10 @@ int stm32_clock_control_init(const struct device *dev)
 	LL_RCC_MSI_SetCalibTrimming(0);
 
 #if STM32_MSI_PLL_MODE
+
```

Comment:
I understand that LSE is required for auto calibration of MSI.
But isn't it possible to use PLL with MSI without auto calibration ?

Reply:
That's actually not the case.
This specific option is to request MSI hardware auto-calibration (setting RCC_CR_MSIPLLEN bit).
So this is not related with the fact of using PLL with MSI, but purely about auto calibration (so-called, but confusing MSIPLL mode)